.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules/
 dist/
 npm-debug.log
+esm/
```

Comment:
Why is this needed?

Reply:
`npm run build` generates this folder

Reply:
It doesn't on `master`, and it shouldn't generate this folder. Looks like the update breaks something. What's in there?

Reply:
Hm, might have been a local issue. I will check tomorrow.

Reply:
Fixed.